recipes/sagemath-db-polytopes/meta.yaml

```diff
@@ -0,0 +1,28 @@
+{% set version = "20120220" %}
+{% set name = "sagemath-db-polytopes" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.bz2
+  url: http://files.sagemath.org/spkg/upstream/polytopes_db/polytopes_db-{{ version }}.tar.bz2
+  sha256: a88d4a4ff1ccb5bb82daf18f8e80c21391a2977ff27855a5f1a3005e8bcbba40
+
+build:
+  number: 0
+  skip: true  # [win]
+
+test:
+  commands:
+    - test -f $PREFIX/share/reflexive_polytopes/reflexive_polytopes_2d
+
+about:
+  home: https://github.com/sagemath/sage
+  license: GPLv2+
+  summary: This package includes lists of 2- and 3-dimensional reflexive polytopes.
+
+extra:
+  recipe-maintainers:
+    - isuruf
```

Comment:
Yes, I already sent an email to the mailing list. I'll send an update and ask for maintainers